src/GraphQL/Mutation/MutationType.php

```diff
@@ -324,14 +327,21 @@ public function buildDataObjectMutations(&$config = [], $context = [])
                 $deleteField = [
                     'type' => $deleteResultType,
                     'args' => [
-                        'id' => ['type' => Type::nonNull(Type::int())],
-                    ], 'resolve' => static function ($value, $args, $context, ResolveInfo $info) use ($entity, $modelFactory, $me) {
+                        'id' => ['type' => Type::int()],
+                        'fullpath' => ['type' => Type::string()],
```

Comment:
@weisswurstkanone is it good practice to define all anonymous function as static and pass the $this pointer as $me? As $this can be accessed within the anonymous function "for free" by just not mark it as static, right?